code/datums/wires/wires.dm

```diff
@@ -14,6 +14,10 @@
 /atom
 	var/datum/wires/wires = null
 
+/atom/proc/attempt_wire_interaction(mob/user)

```

Comment:
Sanity check should probably be here instead

Reply:
Done